Cargo.toml

```diff
@@ -35,4 +36,4 @@ pretty_assertions = "0.5.1"
 
 [build-dependencies]
 fs-utils = "1.0"
-bindgen = "0.35.0"
+bindgen = "0.36.0"
```

Comment:
Without the trailing zero, just `0.36`? Then we automatically get patch releases.